app/build.gradle

```diff
@@ -40,6 +40,6 @@ dependencies {
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.4.0'
     implementation 'tech.gusavila92:java-android-websocket-client:1.2.2'
     implementation fileTree(dir: 'libs', include: ['*.jar'])
-    implementation 'com.github.Splendor-Mobile-Game:ServerSide:develop-SNAPSHOT'
+    implementation 'com.github.Splendor-Mobile-Game:ServerSide:allow-client-to-use-rooms-SNAPSHOT'
```

Comment:
When we merge it to the develop it won't work. Please change back to develop-SNAPSHOT

Reply:
fixed